src/web/mjs/connectors/Team1x1.mjs

```diff
@@ -10,74 +10,58 @@ export default class Team1x1 extends Connector {
+    async _getMangas() {
         let request = new Request( this.url + '/manga/', this.requestOptions );
-        this.fetchDOM( request, 'div.pagination div.wp-pagenavi > a.page:last-of-type' )
-            .then( data => {
-                let pageCount = parseInt( data[0].text.trim() );
-                let pageLinks = [... new Array( pageCount ).keys()].map( page => request.url + 'page/' + ( page + 1 ) + '/' );
-                return this._getMangaListFromPages( pageLinks );
-            } )
-            .then( data => {
-                callback( null, data );
-            } )
-            .catch( error => {
-                console.error( error, this );
-                callback( error, undefined );
-            } );
+        let data = await this.fetchDOM( request, 'div.pagination div.wp-pagenavi > a.last' );
+        let pageCount = parseInt( data[0].href.split("/").slice(-2, -1)[0].trim() );
+        let pageLinks = [... new Array( pageCount ).keys()].map( page => request.url + 'page/' + ( page + 1 ) + '/' );
+        return await this._getMangaListFromPages( pageLinks );
     }
 
-    _getChapterList( manga, callback ) {
+    async _getChapters( manga ) {
         let request = new Request( this.url + manga.id, this.requestOptions );
-        this.fetchDOM( request, 'div.single-manga-chapter div.container div.row div.col-md-12 a' )
-            .then( data => {
-                let chapterList = data
-                    .filter( element => element.href.startsWith( this.url ) )
-                    .map( element => {
-                        return {
-                            id: this.getRootRelativeOrAbsoluteLink( element, request.url ),
-                            title: element.text.replace( manga.title, '' ).trim(),
-                            language: ''
-                        };
-                    } );
-                callback( null, chapterList );
-            } )
-            .catch( error => {
-                console.error( error, manga );
-                callback( error, undefined );
+        let data = await this.fetchDOM( request, 'div.single-manga-chapter div.container div.row div.col-md-12 a' );
+        return data
+            .filter( element => element.href.startsWith( this.url ) )
+            .map( element => {
+                return {
+                    id: this.getRootRelativeOrAbsoluteLink( element, request.url ),
+                    title: element.text.replace( manga.title, '' ).trim(),
+                    language: ''
+                };
             } );
     }
 
-    _getPageList( manga, chapter, callback ) {
-        let script = `new Promise( resolve => resolve( PushManga.pages.original.map( p => p.page + '?token=' + p.token ) ) );`;
+    async _getPages(chapter) {
+        let script = `
+        new Promise(resolve => {
+            var interval = setInterval(function() {
+                if (typeof PushManga !== "undefined") {
+                    clearInterval(interval);
```

Comment:
No need to clear interva;, promises can only be resolve/reject once and browser window will be closed anyway.